packages/twenty-front/src/modules/support/components/__stories__/SupportChat.stories.tsx

```diff
@@ -48,5 +48,8 @@ export const Default: Story = {
   play: async () => {
     const canvas = within(document.body);
     expect(await canvas.findByText('Support')).toBeInTheDocument();
+    await userEvent.click(canvas.getByText('Support'));
```

Comment:
this looks more like a SupportDropdown test than a SupportChat test, we would need both!